test/libsolidity/syntaxTests/structs/member_type_func.sol

```diff
@@ -4,4 +4,4 @@ contract C {
   function g(function(S memory) external) public {}
 }
 // ----
-// TypeError 5172: (50-51): Name has to refer to a struct, enum or contract.
+// TypeError 5172: (50-51): Name has to refer to a user-defined value type, struct, enum or contract.
```

Comment:
@wechman Should I revert these as well, or just stick to the superfluous `.`?

Reply:
Please stick to the superfluous `.`.